### DISCUSSION TITLE
How to get the fuel supply train itself to refuel

### DISCUSSION THREAD

**Discussion by bwoods89** [Opinionated-Blueprints/10-Books-Full-of-Rails#48] [General]
I've reviewed the wiki about refueling, but I can't figure out how the fuel supply train itself is supposed to get fuel. I don't assign it a group like the other trains, so it won't just go to a refueling station and fuel up. I don't see a BP that looks like it's meant to refuel the fuel supply train either. Do I just need to manually run a belt of fuel to the train's engine myself?

**Comment by Bocian-1** [comment]
At first, you need to give it some fuel by hand. Afterwards, it will be felled at the refuelling station during supply runs. You can do the fuelling at the fuel supply station as well if you want.

**Reply by bwoods89** [reply]
I'll have to watch it now and see how it works. I imagine my issue came from trying to make the refueling station work before I had unlocked selector combinators

**Reply by Bocian-1** [reply]
oh that's definitely not going to happen without the selector

**Reply by bwoods89** [reply]
I tried wiring in a constant combinator that was sending a coal signal of 1, but I must have missed one or more of the connections the selector combinator had.
